### DISCUSSION TITLE
Feature Request: Subscription File Location

### DISCUSSION THREAD

**Discussion by afortaleza** [FreeTubeApp/FreeTube#2157] [General]
I host my subscription file in OneDrive but I always have to Export/Import manually. It would be good if we could configure the subscription file location, this way I could point FreeTube to read/write subscription file from my local OneDrive and let the OneDrive sync do the rest.

The same is valid for all FreeTube settings, if we could configure where to read the other settings from we could use OneDrive or any other similar tool to sync and easily restore and never lose subscriptions.

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Ur request is a duplicate of https://github.com/FreeTubeApp/FreeTube/issues/370
